### ISSUE TITLE
Setting ALGOLIA_APPLICATION_ID doesn't seem to work

### ISSUE THREAD

**Issue by mvxt** [algolia/setup-algolia-cli#4]
I've got my job step configured like this:

```yaml
    - uses: algolia/setup-algolia-cli@master
      env:
        ALGOLIA_APPLICATION_ID: ${{ secrets.ALGOLIA_APP_ID }}
        ALGOLIA_ADMIN_API_KEY: ${{ secrets.ALGOLIA_ADMIN_API_KEY }}
    - run: algolia indices clear ${{ vars.ALGOLIA_DEV_INDEX }}
    - run: algolia objects import ${{ vars.ALGOLIA_DEV_INDEX }} -F dist/docs/search.json --auto-generate-object-id-if-not-exist
```

Despite setting the `ALGOLIA_APPLICATION_ID` and the `ALGOLIA_ADMIN_API_KEY`, it fails with this error:
```
Run algolia indices clear dev_fire_docs
  algolia indices clear dev_fire_docs
  shell: /usr/bin/bash -e {0}
Authentication error: you have not configured your Application ID yet
Please run `algolia profile add` to configure your first profile.
Error: Process completed with exit code [4](https://github.com/firehydrant/fh-docs/actions/runs/4258172538/jobs/7409119162#step:7:5).
```

It seems odd that I'd still need to configure a profile after specifying both the app ID and the API key. Either that isn't working, or the instructions need to be updated.

Is there something obvious I'm not doing correctly?

**Comment by northern-mindset** [comment]
It seems that the code does nothing with these environment variables: https://github.com/algolia/setup-algolia-cli/search?q=ALGOLIA_APPLICATION_ID&type=code

The code only downloads and extracts the binary.

Moreover, in the [official CLI documentation](https://www.algolia.com/doc/tools/cli/get-started/authentication/#authentication-without-profiles) it is recommended:

> If you don’t want to create profiles, you can use the `--application-id` and `--admin-api-key` options for every command. This can be useful if you want to store your Algolia credentials elsewhere and reference them using environment variables—for example, in continuous deployment.

**Comment by northern-mindset** [comment]
From the example in [another Algolia repo](https://github.com/algolia/cli-github-action-example) it becomes clear where to specify these env variables. It's for the CLI itself, not for the `setup-algolia-cli` action:

https://github.com/algolia/cli-github-action-example/blob/b41f3d808806af8e5312b3982f82067fe3224182/.github/workflows/deploy-settings.yml#L22-L25
```yaml
        run: algolia settings import ${{github.ref_name}}_${{secrets.ALGOLIA_INDEX_NAME}} -F deploy/settings.json
        env:
          ALGOLIA_APPLICATION_ID: ${{ secrets.ALGOLIA_APPLICATION_ID }}
          ALGOLIA_ADMIN_API_KEY: ${{ secrets.ALGOLIA_API_KEY }}
```

The docs here should be fixed.

**Comment by clemfromspace** [comment]
Fixed in 67edb64. Thanks @mvxt for your issue and @anantakrishna for the solution (and my apologies for the late reply).
